include/avif/avif.h

```diff
@@ -85,8 +85,11 @@ extern "C" {
     ((AVIF_VERSION_MAJOR * 1000000) + (AVIF_VERSION_MINOR * 10000) + (AVIF_VERSION_PATCH * 100) + AVIF_VERSION_DEVEL)
 
 typedef int avifBool;
-#define AVIF_TRUE 1
-#define AVIF_FALSE 0
+enum avifTrueFalse
+{
+    AVIF_FALSE = 0,
```

Comment:
MSVC emits a warning:

D:\a\libavif\libavif\src\read.c(5482): warning C4127: conditional expression is constant

src/read.c:5482 is:

```
            AVIF_ASSERT_OR_RETURN(AVIF_FALSE);
```

It is strange that MSVC does not warn if the conditional expression is the integer literal 0, which is also a constant.

We can try changing that line to `AVIF_ASSERT_OR_RETURN(0);` or adding a new macro `AVIF_ASSERT_NOT_REACHED_OR_RETURN` (or `AVIF_NOT_REACHED_OR_RETURN`).